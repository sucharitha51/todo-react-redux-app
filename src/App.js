import React, { Component } from 'react';
import logo from './logo.svg';

//Import the modified App.css
import './App.css';

// Import the Routes component, which contains our Route setup

import { Routes } from './Routes'


// Provider component is a react-redux component for setting up the Redux Store

import { Provider } from 'react-redux'

// Import the ConfigureStore that holds the initial Configuration

import { configureStore } from './store/configureStore'

// Create a Store from the Configuration, we can pass a Initial State here
const store = configureStore()

const App = (props) => {
  return (
      <Provider store={store} >
        <Routes />
      </Provider>
  )
}
export default App;
