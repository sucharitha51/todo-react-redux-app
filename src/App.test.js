import React from 'react';
import { configure, shallow } from 'enzyme'  
import ReactDOM from 'react-dom';
import App from './App';
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })

let wrapper

beforeEach(() => {
  wrapper = shallow(<App />)
})

describe('App  component', () => {

  it('App renders without crashing', () => {
    expect(wrapper.exists()).toEqual(true);
  });

  it('App should match with snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
})