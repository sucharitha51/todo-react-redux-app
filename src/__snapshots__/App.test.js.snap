// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App  component App should match with snapshot 1`] = `"<header><h1>Todo List</h1><div class=\\"actions\\"><span class=\\"add-new\\">Add Todo Item</span><a href=\\"/\\" style=\\"text-decoration:none;color:#000000\\" class=\\"add-new\\">Reset All Todos</a></div></header><div class=\\"todos-list\\"><div class=\\"columns\\"><p class=\\"title\\">Title</p><p class=\\"priority\\">Priority</p><p class=\\"date\\">date</p><p class=\\"isDone\\">Done</p><p class=\\"isDone\\">View</p></div><p>No data.</p></div>"`;
